### PR TITLE
Improve Import Perf

### PR DESCRIPTION
**Pull request by tabascq**
Improve the perf of import such that it succeeds by omitting file copy, and provide user instructions for how to complete the job by clining the container in Azure Storage Explorer.

The model for the container display is very kludgy; I tried to do it the "right" way by creating a separate model item within Import.cshtml.cs, but SelectList was crashing internally and not explaining why. Apologies!

### CODE REVIEW

**Comment by morganbr** [comment]
Do we also need #624?

**Comment by morganbr** [comment]
LGTM, but I seem to have lost approval permissions. I'll look into it.

**Review by morganbr — APPROVED** [review]

**Comment by tabascq** [comment]
LOL, I completely forgot about #624 existing at all. I'd been holding off on this until I added instructions as to how to perform the container clone and assumed I had never put it up for review.

I ran this code to perform today's clone and it took maybe 5 seconds without bumping the server, so I don't think the rest is necessary. I'll just close that one.

Thanks for the review!

**Pull request merged**
